Makefile

```diff
@@ -10,15 +10,15 @@ dev-setup:
 
 .PHONY: tests ## Run unit tests
 tests:
-	py.test graphene_django --cov=graphene_django -vv
+	pytest graphene_django --cov=graphene_django -vv
 
 .PHONY: format ## Format code
 format:
-	black graphene_django examples setup.py
+	tox -e pre-commit
```

Comment:
I think this actually has a different effect, since it won't apply to all existing files and will just apply to uncommitted changes. Perhaps this should be `pre-commit run --all-files` instead of just `pre-commit`?

Reply:
ah this runs the `pre-commit` tox env, not the pre-commit command. In `tox.ini` you will see https://github.com/graphql-python/graphene-django/blob/7bd4a684b243c9bd61d015ef4a4ce651a93bbc01/tox.ini#L37-L41

that's the command being run

but I agree on adding back black and reversing this change.

Reply:
Ah gotcha, okay 👍